reddit-background

```diff
@@ -139,10 +147,14 @@ class Desktop(object):
 
         random.shuffle(image_urls)
 
-        for image_url in image_urls:
+        log('Number of images to download: {0}'.format(NUMBER_OF_IMAGES))
+        number_downloaded = 0
+        while (number_downloaded < NUMBER_OF_IMAGES):
```

Comment:
The user may have set `--number-of-images` to be greater than the number of candidates images available. We should probably handle that.

If that's the case, we should have the function raise a proper exception, that we then catch to print out an error message to stderr.